### ISSUE TITLE
Feature Request: Use set files to validate custom cards in cubes

### ISSUE THREAD

**Issue by IanOlds** [dr4fters/dr4ft#1581]
The feature of being able to upload a custom set that includes custom cards it really nice, however I do not see a way for cubes to use this. If you implemented this, you should be able to select a set from a drop down, and if you type in a custom card name into the cube box, dr4ft would recognize it as a valid card because it is in the set file, even if it is not a real card. Then, there can be more information including images pulled and used for the draft.

TLDR: cubes should be able to use set files for validating cards for use in the cube.
